### PR TITLE
consistency and style fix

### PR DESCRIPTION
**Pull request by Mashimiao**
replace file sytem with filesystem for consistent in context
keep one sentence per line

Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -58,13 +58,14 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
-  * Windows: the type of file system on the volume, e.g. "ntfs".
+  * Windows: the type of filesystem on the volume, e.g. "ntfs".
   * Solaris: corresponds to "type" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`source`** (string, OPTIONAL) A device name, but can also be a directory name or a dummy.
   * Windows: the volume name that is the target of the mount point, \\?\Volume\{GUID}\ (on Windows source is called target).
   * Solaris: corresponds to "special" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`options`** (list of strings, OPTIONAL) Mount options of the filesystem to be used.
-  * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
+  * Linux: supported options are listed in the [mount(8)][mount.8] man page.
+  Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
```
Comment (wking) [inline config.md]: I think you want deeper indenting here to show that this is part of the `* Linux: …` entry.  For an example with a two-space indent, see [here][1], although that's getting cleaned up to use four-space indents (which Pandoc likes better) in #724.

[1]: https://github.com/opencontainers/runtime-spec/blame/45c3fd4b5002bc54786b85a19b1a2105502e29a6/config.md#L358-L359

File: config.md [hunk 2]
```diff
@@ -132,8 +133,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
-  capabilities contains the following properties:
+* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container.
+  Valid values are platform-specific.
+  For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`.
+  Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
+  `capabilities` contains the following properties:
```
Comment (wking) [inline config.md]: `capabilities` is already using four-space indents (in the list below).  Can we use them here while we're touching the lines?

File: config.md [hunk 3]
```diff
@@ -144,7 +148,8 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
 
     * **`type`** (string, REQUIRED) - the platform resource being limited, for example on Linux as defined in the [setrlimit(2)][setrlimit.2] man page.
     * **`soft`** (uint64, REQUIRED) - the value of the limit enforced for the corresponding resource.
-    * **`hard`** (uint64, REQUIRED) - the ceiling for the soft limit that could be set by an unprivileged process. Only a privileged process (e.g. under Linux: one with the CAP_SYS_RESOURCE capability) can raise a hard limit.
+    * **`hard`** (uint64, REQUIRED) - the ceiling for the soft limit that could be set by an unprivileged process.
+    Only a privileged process (e.g. under Linux: one with the CAP_SYS_RESOURCE capability) can raise a hard limit.
```
Comment (wking) [inline config.md]: Also want deeper indents here to make it clearly part of the `hard` list entry.

File: config.md [hunk 4]
```diff
@@ -302,13 +307,13 @@ For Windows based systems the user structure has the following fields:
 
 **`platform`** (object, REQUIRED) specifies the configuration's target platform.
 
-* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) file system bundle.
+* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) filesystem bundle.
   The runtime MUST generate an error if it does not support the specified **`os`**.
-  Bundles SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
+  Value for **`os`** SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
```
Comment (wking) [inline config.md]: I don't think readers are likely to forget which property we're talking about ;).  Can we shorten this line?  Maybe:

    Values should be, and runtimes SHOULD understand, valid [`GOOS`][go-environment] values.

File: config.md [hunk 5]
```diff
@@ -428,7 +433,7 @@ Keys MUST be unique within this map.
 Keys MUST NOT be an empty string.
 Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
 Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
-Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
+Runtime implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
```
Comment (wking) [inline config.md]: Can we just use “Runtimes”?

```
$ git describe --always
v1.0.0-rc5-148-g45c3fd4
$ git grep -i runtime.implementations *.md | wc -l
2
$ git grep -i runtimes *.md | wc -l
19
```

**Comment by Mashimiao** [comment]
all comments have been updated.  PTAL

File: config.md [hunk 6]
```diff
@@ -132,8 +133,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
-  capabilities contains the following properties:
+* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container.
+  Valid values are platform-specific.
+  For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`.
+  Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
```
Comment (wking) [inline config.md]: When I asked for four-space indents [here][1], I meant all the lines after `* **capabilities** …` (sorry for not clarifying).  So it should be:

```
* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container.
    Valid values are platform-specific.
    For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`.
    Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
    `capabilities` contains the following properties:
```

[1]: https://github.com/opencontainers/runtime-spec/pull/811#discussion_r116292394

Comment (Mashimiao) [inline config.md]: updated, but there are a lot of sentences do not start with four-space indents, will not fix them in this PR.
I can submit a follow-PR to format all of them.

File: config.md [hunk 7]
```diff
@@ -323,7 +328,7 @@ For Windows based systems the user structure has the following fields:
 ## <a name="configPlatformSpecificConfiguration" />Platform-specific configuration
 
 [**`platform.os`**](#platform) is used to specify platform-specific configuration.
-Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Runtimes MAY support any valid values for platform-specific fields as part of this configuration.
```
Comment (TomSweeneyRedHat) [inline config.md]: Fwiw, I think "Runtime implementations" is better than 'Runtimes" and I'd change the three 'Runtimes' below to that too.  My $.02, your call to make.

File: config.md [hunk 8]
```diff
@@ -428,7 +433,7 @@ Keys MUST be unique within this map.
 Keys MUST NOT be an empty string.
 Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
 Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
-Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
+Runtimes MUST NOT generate an error if they encounter an unknown annotation key when reading/processing this configuration file.
```
Comment (TomSweeneyRedHat) [inline config.md]: Suggest 'reading or processing' here and below.  I'm not a big fan of '/' over 'or'.  Again, my $.02, your call.

File: config.md (hunk 7 above)
Comment (wking) [inline config.md]: > ... I'd change the three 'Runtimes' below to that too.

Would you update [the other instances][1] (beyond those already touched by this PR) too?

Can you put a finger on why you likw including "implementations" better?  A bare "runtimes" is shorter, and I can't think of a non-implementation runtime.

[1] :https://github.com/opencontainers/runtime-spec/pull/811#pullrequestreview-38088236

**Comment by Mashimiao** [comment]
rebased. @opencontainers/runtime-spec-maintainers  PTAL

**Comment by Mashimiao** [comment]
rebased around #846

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**